py/bitbox02/bitbox02/bitbox02/bitbox02.py

```diff
@@ -788,7 +788,13 @@ def eth_pub(
         )
         return self._eth_msg_query(request, expected_response="pub").pub.pub
 
-    def eth_sign(self, transaction: bytes, keypath: Sequence[int], chain_id: int = 1) -> bytes:
+    def eth_sign(
+        self,
+        transaction: bytes,
+        keypath: Sequence[int],
+        address_case: eth.ETHAddressCase.ValueType = eth.ETH_ADDRESS_CASE_MIXED,
```

Comment:
sorry I forgot: also add an entry to /py/bitbox02/CHANGELOG.md, as this changes the Python API.